extensions/blocks/map/edit.js

```diff
@@ -449,7 +450,7 @@ class MapEdit extends Component {
 								ref={ this.mapRef }
 								scrollToZoom={ allowScrollToZoom }
 								showFullscreenButton={ showFullscreenButton }
-								mapStyle={ mapStyle }
+								mapStyle={ mapStyle ? mapStyle.name : 'default' }
```

Comment:
We seem to only use `mapStyle` as a string, so we could we defaulth this when it's set and leave this line as is?

Reply:
`mapStyle.preview` is used for the `placholderPreview`. That said, I think I'm going to redo how it's storing the style, so it works the same way as prior to this PR. We can just ignore the custom class that Core will add to the block.

Reply:
Turned out to be relatively easy, done in e2927a623.